docs/docs/models-usage.md

```diff
@@ -540,7 +540,12 @@ This will produce an outer join. However, a `where` clause on a related model wi
 User.findAll({
   include: [
     {model: Tool, as: 'Instruments', include: [
-      {model: Teacher, include: [ /* etc */], required: false}
+      {
```

Comment:
The closing `}` should be on a seperate line and the items of the object should be indented